torch/_inductor/config.py

```diff
@@ -165,6 +165,11 @@ class triton:
     # Use cudagraphs on output code
     cudagraphs = False
 
+    # Use cudagraph trees for memory pooling if `cudagraphs` is True
+    cudagraph_trees = True
+
+    debug_cudagraph_trees = True
```

Comment:
True by default?

Reply:
it's not doing any logging, it's just adding runtime checks, which I think we should have on initiially